Tasks/UseRubyVersionV0/userubyversion.ts

```diff
@@ -41,11 +41,7 @@ export async function useRubyVersion(parameters: TaskParameters, platform: Platf
     if (platform !== Platform.Windows) {
         // replace the default
         const dest: string = '/usr/bin/ruby';
-        if (fs.existsSync(dest)) {
-            task.debug('removing ' + dest);
-            fs.unlinkSync(dest);
-        }
-        fs.symlinkSync(path.join(toolPath, 'ruby'), dest);
+        task.execSync('sudo', `ln -sf ${path.join(toolPath)} ${dest}`); // replace any existing
```

Comment:
Any reason for calling `path.join` with a single argument?

Reply:
Yeah, because it's a bug. :(

Thanks!